src/components/NewComment.jsx

```diff
@@ -34,6 +34,7 @@ const NewComment = ({
         ...comments,
         {
           body: inputComment,
+          author: username,
```

Comment:
This could be done using useContext, rather than prop drilling. Have a look at the notes!